doc/nrf/app_dev/device_guides/thingy91x/thingy91x_recover_to_factory_firmware.rst

```diff
@@ -0,0 +1,64 @@
+.. _thingy91x_recover_to_factory_firmware:
+
+Recovering the Thingy:91 X to factory firmware
+##############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can recover the Thingy:91 X to factory firmware using the ``nrfutil device`` command.
```

Comment:
This guide is entirely about using nRF Util. This should be covered in this page, and not in the NCS: https://docs.nordicsemi.com/bundle/nrfutil/page/nrfutil-device/guides/programming_recovery.html

Reply:
The reason to move this guide from the [add-on page](https://docs.nordicsemi.com/bundle/addon-nrfcloud_firmware-latest/page/troubleshooting.html#factory_reset_j-link) to the developing with guide was because it was difficult for the user to find the doc when they need to. Pacing it under developing with guide would be easy to find.

cc: @maxd-nordic

Reply:
The user guide is all about using nRF Util. Placing it under nRF Util docs makes most sense. It is also easier to maintain there if anything in nRF Util implementation changes. You can link to this one source of truth later from both the add-on docs and the NCS docs.

Reply:
Then, maybe keep the guide under "developing with" for now and move the guide to nRF util in a separate task? Will create a Jira under -- NCSIDB for you.